src/jobflow_remote/jobs/daemon.py

```diff
@@ -169,6 +177,12 @@ def sock_filepath(self) -> Path:
             raise DaemonError(msg)
         return path
 
+    @lazy_property
+    def job_controller(self):
+        if self._job_controller is None:
+            return JobController.from_project(self.project)
+        return self._job_controller
```

Comment:
Actually not because the job_controller is optional in the init. If it has been passed to init, it should be that one, if not then it should be instantiated with from_project. Now maybe it's not useful to be have it in the init ... we could maybe always use the property. What do you think ?

Reply:
ok, then I think you can remove the lazy_property, becase in any case you are already reusing the one that is in `self._job_controller`

Reply:
Ok then in that case I'd just set the _job_controller in the if and return it after.

def job_controller(self):
        if self._job_controller is None:
            self._job_controller = JobController.from_project(self.project)
        return self._job_controller

Like this